paymentsheet/src/test/java/com/stripe/android/paymentsheet/verticalmode/PaymentMethodRowButtonCheckmarkScreenshotTest.kt

```diff
@@ -0,0 +1,143 @@
+package com.stripe.android.paymentsheet.verticalmode
+
+import androidx.compose.foundation.Image
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.padding
+import androidx.compose.material.Text
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.unit.dp
+import com.stripe.android.screenshottesting.FontSize
+import com.stripe.android.screenshottesting.PaparazziRule
+import com.stripe.android.ui.core.R
+import org.junit.Rule
+import org.junit.Test
+
+internal class PaymentMethodRowButtonCheckmarkScreenshotTest {
+
+    @get:Rule
+    val paparazziRule = PaparazziRule(
+        FontSize.entries,
+        boxModifier = Modifier
+            .padding(16.dp)
+            .fillMaxWidth()
+    )
+
+    @Test
+    fun testInitialState() {
+        paparazziRule.snapshot {
+            PaymentMethodRowButtonCheckmark(
+                isEnabled = true,
+                isSelected = false,
+                iconContent = {
+                    Image(
+                        painter = painterResource(id = R.drawable.stripe_ic_paymentsheet_pm_card),
+                        contentDescription = null
+                    )
+                },
+                title = "**** 4242",
+                subtitle = null,
+                onClick = {},
+            )
+        }
+    }
+
+    @Test
+    fun testDisabledState() {
+        paparazziRule.snapshot {
+            PaymentMethodRowButtonCheckmark(
+                isEnabled = false,
+                isSelected = false,
+                iconContent = {
+                    Image(
```

Comment:
I thought we'd have a generic inner content item, and that would be used for all types. This makes it feel like you would have to change 3 places if you wanted to add something new to the inner content.